src/components/VmDetails/CardEditButton.js

```diff
@@ -37,12 +39,28 @@ class CardEditButton extends React.Component {
   }
 
   render () {
-    const { tooltip, id } = this.props
+    const { tooltip, id, editable, disableTooltip } = this.props
     const { editEnabled } = this.state
 
     const classes = `${style['card-edit-button']} ${style[editEnabled ? 'card-edit-button-enabled' : 'card-edit-button-disabled']}`
     const myClick = editEnabled ? noop : this.enableEditHandler
 
+    if (!editable && disableTooltip) {
+      return <OverlayTrigger
+        overlay={<Tooltip><span>{disableTooltip}</span></Tooltip>}
+        placement='right'
+        trigger={['hover', 'focus']}
+        rootClose={false}
+      >
+        <a className={`${style['card-edit-button']} ${style['card-edit-button-disabled']}`} id={id}>
+          <Icon type='pf' name='edit' />
+        </a>
+      </OverlayTrigger>
```

Comment:
Thought: Could just use the `OverlayTrigger` tooltip for the normal tooltip case so the "normal" and the "edit is disabled" tooltips look the same.

Reply:
Yes, it will be good for consistency, but I think this is for separate issue, because this PR is not about tooltips.

Reply:
No, but you added one.  Why not add it to be consistent?  It's not a bug fix PR, it is a redesign.

Reply:
See #1021.

I still don't think it is necessary or even reasonable to have to generate a separate issue like this, but in the interest of getting this PR to QE, #1021 was created.